sql/fileKeywordsSchemaTriggers.sql

```diff
@@ -0,0 +1,85 @@
+-- create table for the file keywords
+CREATE TABLE `file_keyword` (
+  `fid` int(11) NOT NULL,
+  `keyword` varchar(128) NOT NULL,
+  PRIMARY KEY (`fid`,`keyword`),
+  CONSTRAINT `fk_file_keyword_file` FOREIGN KEY (`fid`) REFERENCES `file` (`fid`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- add file attributes to simplify searches
```

Comment:
this could be kept but basicly is only relevant once as we only have one instance of springfiles